themes/_core/js/product.js

```diff
@@ -154,6 +154,7 @@ function updateProduct(event, eventType, updateUrl) {
       url: updateUrl + ((updateUrl.indexOf('?') === -1) ? '?' : '&') + formSerialized + preview,
       method: 'POST',
       data: {
+        quickview: parseInt($('.modal.quickview.in').length),
```

Comment:
Suggestion: if we're going to introduce a `quickview` boolean both in PHP and in JS, maybe we can make it 100% unambiguous ?

This statement basically says "OK, if selector `'.modal.quickview.in'` does exist, then I know that I am in a quickview" so it's trying to find out whether or not it's a quickview using the HTML markup.

What about making it 100% clear ? I'm suggesting to put somewhere in the HTML markup a data attribute `is-quickview`. So the code would become

```suggestion
        quickview: $('my-div-with-my-informations').data('is-quickview')
```

Reply:
Hey, once again thanks for the suggestion, but as I said on Slack, I don't think that we would ever had different element with this kind of class in our shop, and thanks to checking it like this theme developers don't need to do anything to make them compatible with my solution